### ISSUE TITLE
Sticky nav/ad makes page jump

### ISSUE THREAD

**Issue by OliverJAsh** [guardian/frontend#10147]
Not sure if this is related to the sticky nav and/or ad, but I presume it's happening when the JS for the sticky situation is executed. I tested on CODE with it disabled and it no longer jumped.

On a side note, I think one of our "design lines in the sand" should be no page jumping.

![untitled](https://cloud.githubusercontent.com/assets/921609/9253066/d3fa0d6e-41d3-11e5-8388-fb3c08b23ccf.gif)

**Comment by paperboyo** [comment]
> one of our "design lines in the sand" should be no page jumping

+1!

(https://github.com/guardian/frontend/issues/3074#issuecomment-34761610, https://github.com/guardian/frontend/issues/8444#issuecomment-76284160 and many, many more...)

**Comment by ScottPainterGNM** [comment]
Having discussed with @stephanfowler - as we will reimplement mobile before it coming back on, this bug is closed for now but needs to be considered for when we revisit this.
